typespec/Applications.Core/environments.tsp

```diff
@@ -103,24 +110,47 @@ model SecretConfig {
   secret?: string;
 }
 
-@doc("The Cloud providers configuration")
+#suppress "@azure-tools/typespec-azure-core/bad-record-type"
+@doc("ProviderConfigProperties specifies provider configuration details needed for recipes.")
```

Comment:
The pattern of struct name (`ProviderConfigProperties` in this case) followed by description is a convention in go struct comments, but not in the typespec. Let's be consistent with the format used by rest of the typespec descriptions in the Radius repo.